### PR TITLE
Display pod tolerations with kubectl describe pod

### PR DESCRIPTION
**Pull request by aveshagarwal**
@kubernetes/rh-cluster-infra @kubernetes/kubectl

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/30381)

### CODE REVIEW

**Comment by fabianofranz** [comment]
@aveshagarwal can you please paste a sample of the output?

**Comment by aveshagarwal** [comment]
With this patch:

```
Name:		dapi-test-pod
Namespace:	default
Node:		127.0.0.1/127.0.0.1
Start Time:	Wed, 10 Aug 2016 13:11:03 -0400
Labels:		<none>
Tolerations:	key2=value1:Equal
		        key3=value2:NoSchedule
		        key4=value4:Equal:PreferNoSchedule
Status:		Running

```

**Comment by fabianofranz** [comment]
@aveshagarwal Thanks! Overall looks good, but I'd put it lower in the describe screen, probably after the `QoS` information (before `Events`) or something like that. Also I'd suggest to add a test in describe_test.go, specially because tolerations is not yet completely stable (there's https://github.com/kubernetes/kubernetes/issues/25320).

**Comment by aveshagarwal** [comment]
@fabianofranz addressed comments. PTAL.

**Comment by fabianofranz** [comment]
LGTM, @smarterclayton can you please tag?

File: pkg/kubectl/describe.go [hunk 1]
```diff
@@ -2474,3 +2475,49 @@ func printTaintsMultilineWithIndent(out io.Writer, initialIndent, title, innerIn
...
+	for i, key := range keys {
+		for _, toleration := range tolerations {
+			if toleration.Key == key {
+				if i != 0 {
+					fmt.Fprint(out, initialIndent)
+					fmt.Fprint(out, innerIndent)
+				}
+				fmt.Fprintf(out, "%s=%s", toleration.Key, toleration.Value)
+				if len(toleration.Operator) != 0 {
+					fmt.Fprintf(out, ":%s", toleration.Operator)
```
Comment (smarterclayton) [inline pkg/kubectl/describe.go]: Why are we using `:` to separate here?  Any particular reason or comparable behavior from elsewhere in our code?

Comment (aveshagarwal) [inline pkg/kubectl/describe.go]: for taints, the output format is : `%s=%s:%s` for key=value:effect  
So I just extended it to toleration: `%s=%s:%s:%s` for key=value:operator:effect

**Pull request merged**